frontend/libs/erxes-ui/src/modules/record-field/components/NumberField.tsx

```
@@ -0,0 +1,64 @@
+  }
+>(
+  (
+    { placeholder, value, scope, onSave, onValueChange, children, ...props },
+    ref,
+  ) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [editingValue, setEditingValue] = useState(value);
+
+    const handleAction = () => {
+      if (editingValue === value) return;
+      onSave && onSave(editingValue);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)